### PR TITLE
Add omnibus_embedding_pairwise to pipeline module

### PR DESCRIPTION
**Pull request by Nyecarr**
- [ ] Does this PR add any new dependencies?
- [ ] Does this PR modify any existing APIs?
   - [ ] Is the change to the API backwards compatible?
- [ ] Have you built the documentation (reference and/or tutorial) and verified the generated documentation is appropriate?

#### What does this implement/fix? Briefly explain your changes.
Add wrapper api to omnibus and add the pairwise omni technique used in `topologic`

#### Any other comments?
Added a couple elbow cut tests for pipeline ase/lse just for peace of mind.

### CODE REVIEW

File: graspologic/pipeline/embed/omnibus_embedding.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from beartype import beartype
+
+
+@beartype
+def omnibus_embedding():
+    raise NotImplementedError()
```
Comment (daxpryce) [inline graspologic/pipeline/embed/omnibus_embedding.py:9]: 👀

File: graspologic/pipeline/embed/omnibus_embedding.py [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+            right = right[:, :column_index]
+            embedding = np.concatenate((left, right), axis=1)
+        else:
+            embedding = embedding[:, :column_index]
+
+    return embedding
+
+
+def _augment_graph(graph, node_ids, weight_attribute):
+    graph_as_array = nx.to_numpy_array(
```
Comment (daxpryce) [inline graspologic/pipeline/embed/omnibus_embedding.py:259]: not sparse?

Comment (Nyecarr) [reply graspologic/pipeline/embed/omnibus_embedding.py:259]: Nope, when #621 pushes through I will make that change.

File: graspologic/pipeline/embed/omnibus_embedding.py [hunk 3]
```diff
@@ -0,0 +1,280 @@
...
+      ----------
+      .. [1] Sussman, D.L., Tang, M., Fishkind, D.E., Priebe, C.E.  "A
+         Consistent Adjacency Spectral Embedding for Stochastic Blockmodel Graphs,"
+         Journal of the American Statistical Association, Vol. 107(499), 2012
+
+      .. [2] Levin, K., Roosta-Khorasani, F., Mahoney, M. W., & Priebe, C. E. (2018).
+          Out-of-sample extension of graph adjacency spectral embedding. PMLR: Proceedings
+          of Machine Learning Research, 80, 2975-2984.
+
+      .. [3] Zhu, M. and Ghodsi, A. (2006). Automatic dimensionality selection from the
```
Comment (daxpryce) [inline graspologic/pipeline/embed/omnibus_embedding.py:123]: we should absolutely reference the paper from https://github.com/microsoft/graspologic/blob/dev/graspologic/embed/omni.py#L132

**Review by daxpryce — APPROVED** [review]

File: graspologic/pipeline/embed/omnibus_embedding.py [hunk 4]
```diff
@@ -0,0 +1,285 @@
...
+    graph_embeddings = []
+
+    # create a graph that contains all nodes and edges across the entire corpus
+    union_graph = graphs[0].copy()
+    for graph in graphs[1:]:
+        union_graph.add_edges_from(graph.edges())
+
+    union_graph_lcc = largest_connected_component(union_graph)
+    union_graph_lcc_nodes = union_graph_lcc.nodes()
+    union_node_ids = np.array(list(union_graph_lcc_nodes))
```
Comment (bdpedigo) [inline graspologic/pipeline/embed/omnibus_embedding.py:161]: just wanted to mention that we have a util function for this here https://github.com/microsoft/graspologic/blob/07c78921491da596e9f438c740805c7a920aaa0d/graspologic/utils/utils.py#L584

Comment (daxpryce) [reply graspologic/pipeline/embed/omnibus_embedding.py:161]: @nyecarr maybe you can use this in #824 too?

**Pull request merged**